### PR TITLE
Italian translation for the Android app.

### PR DESCRIPTION
**Pull request by matteocoder**
If I forgot to properly backslash any apostrophes please let me know!

### CODE REVIEW

**Comment by di72nn** [comment]
```
string 'connectionWizard_providerSelection_provider_wallabagit_name' marked untranslatable but exists in locale 'it'

Apostrophe not preceded by \ (in <b>App wallabag per Android</b><br />wallabag é un'applicazione self hostable che ti permette di non perdere più nessun contenuto. Clicca, salva e leggi quando puoi. Estra il contenuto in modo che tu possa leggere quando puoi.)

Apostrophe not preceded by \ (in Sei sicuro di voler cancellare l'articolo?)

string 'connectionWizard_providerSelection_provider_wallabagit_name' marked untranslatable but exists in locale 'it'

Apostrophe not preceded by \ (in <b>App wallabag per Android</b><br />wallabag é un'applicazione self hostable che ti permette di non perdere più nessun contenuto. Clicca, salva e leggi quando puoi. Estra il contenuto in modo che tu possa leggere quando puoi.)

Apostrophe not preceded by \ (in Sei sicuro di voler cancellare l'articolo?)
```

**Comment by matteocoder** [comment]
Can I merge the pull request now or should I wait for someone else's approval?

File: app/src/main/res/values-it/strings.xml [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+    <string name="connectionWizard_welcome_titleMessage">Benvenuto in wallabag!</string>
+    <string name="connectionWizard_welcome_text">Prima di poter usare l\'app di wallabag devi configurare una connessione. Questo assistente di connessione cercherà di aiutarti.</string>
+    <string name="connectionWizard_welcome_rerunNotice">Puoi comunque riavviare l\'assistente di connessione dalle Impostazioni.</string>
+    <string name="connectionWizard_welcome_prev">Salta</string>
+    <string name="connectionWizard_welcome_next">Successivo</string>
+
+    <string name="connectionWizard_providerSelection_titleMessage">Seleziona il provider di wallabag</string>
+    <string name="connectionWizard_providerSelection_provider_general_name">Altro</string>
+    <string name="connectionWizard_providerSelection_provider_general_desc">Seleziona questa opzione per inserire le impostazioni richieste manualmente</string>
+    <string name="connectionWizard_providerSelection_provider_wallabagit_name" translatable="true">wallabag.it</string>
```
Comment (di72nn) [inline app/src/main/res/values-it/strings.xml]: This line should be removed (`translatable="false"` lines should be present only in the default locale).

**Comment by di72nn** [comment]
Better leave it for somebody else to merge it. Thanks for the translation, BTW.

File: app/src/main/res/values-it/strings.xml [hunk 2]
```diff
@@ -242,7 +242,7 @@
     <string name="connectionWizard_providerSelection_titleMessage">Seleziona il provider di wallabag</string>
     <string name="connectionWizard_providerSelection_provider_general_name">Altro</string>
     <string name="connectionWizard_providerSelection_provider_general_desc">Seleziona questa opzione per inserire le impostazioni richieste manualmente</string>
-    <string name="connectionWizard_providerSelection_provider_wallabagit_name" translatable="true">wallabag.it</string>
+    <string name="connectionWizard_providerSelection_provider_wallabagit_name">wallabag.it</string>
```
Comment (di72nn) [inline app/src/main/res/values-it/strings.xml]: I meant the whole line should be removed.

**Comment by Strubbl** [comment]
@matteocoder can you please squash your commits now, so that we do not pollute the git log too much?

**Comment by matteocoder** [comment]
@Strubbl what do you mean by squashing? Thanks for being patient with me, I'm still learning how GitHub works:)

**Comment by Strubbl** [comment]
No, problem. Thanks for your contribution.

Regarding squashing, basically, it means melting all your 5 commits of this pull request into one commit. [I am quoting @di72nn from another issue](https://github.com/wallabag/android-app/pull/443#issuecomment-284055017) for the details on how to do that:

> I would probably create another branch and git cherry-pick some commits and amend them if needed.
> If you have a plan in mind, you may use git rebase -i to do pretty much the same. You may refer to Git - Rewriting History for some instructions.
> When you're done, git push -f the changes to the same branch.

**Comment by nicosomb** [comment]
Squashed.

**Comment by matteocoder** [comment]
@nicosomb Thanks for squashing them :)

**Pull request merged**